CHANGELOG.md

```diff
@@ -45,10 +39,79 @@ Solidity optimizer: `disabled`
 
 -   `SimulateTxAccessor` - TBD
 
+# Version 1.5.0
+
+## Rename repository
+
+Issue: [#719](https://github.com/safe-global/safe-smart-account/issues/719)
+
+The repository was renamed from `safe-contracts` to `safe-smart-account` to better reflect the purpose of the contracts. Also, the npm package name was changed from `@safe-global/safe-contracts` to `@safe-global/safe-smart-account`.
+
+## Compiler settings
+
+Solidity compiler: [0.7.6](https://github.com/ethereum/solidity/releases/tag/v0.7.6) (for more info see issue [#251](https://github.com/safe-global/safe-smart-account/issues/251))
+
+Solidity optimizer: `disabled`
+
+## Expected addresses with [Safe Singleton Factory](https://github.com/safe-global/safe-singleton-factory)
+
+### Core contracts
+
+-   `Safe` - `0x7097743807D6f33bfCA1AE02492Ddb9F8a62E10A`
```

Comment:
i actually realised that we didn't back merge the 1.4.1 changelog that added zksync addresses: https://github.com/safe-global/safe-smart-account/blob/v1.4.1-3/CHANGELOG.md

(this version is on a separate branch because incremental releases of 1.4.1 and the current upstream diverged too much)

let me know if you want to tackle this or if I should do it

Reply:
Nice catch!

Reply:
Added zksync addresses.

> let me know if you want to tackle this or if I should do it

Not sure I understand what needs to be done. Could you clarify?